backend/app/services/memory/manager.py

```
@@ -0,0 +1,375 @@
+            delete_count = 0
+            error_count = 0
+
+            for memory in memories:
+                memory_id = memory.id
+                try:
+                    success = await self._client.delete_memory(memory_id)
+                    if success:
+                        delete_count += 1
+                    else:
+                        error_count += 1
+                except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)